addons/udes_stock/tests/test_stock_warehouse.py

```diff
@@ -7,22 +7,37 @@ def setUpClass(cls):
         super(TestStockWarehouseModel, cls).setUpClass()
         cls.Warehouse = cls.env["stock.warehouse"]
 
-    def test01_get_test_picking_types(self):
+    def test_get_test_picking_types(self):
         """Test checks if the test picking types are present"""
         test_picking_types = self.warehouse.get_picking_types().filtered(
             lambda pt: "TEST" in pt.name
         )
         self.assertEqual(len(test_picking_types), 6)
 
-    def test02_get_picking_types(self):
+    def test_get_picking_types(self):
         """Test checks if the test picking types and the normal pick types are present"""
         test_picking_types = self.warehouse.get_picking_types()
         self.assertEqual(len(test_picking_types), 9)
 
-    def test03_get_default_picking_types_new_warehouse(self):
+    def test_get_default_picking_types_new_warehouse(self):
         """Create a new warehouse and look at the default picking types"""
         # Create a new company creates a new warehouse
         test_company = self.create_company("test_company")
         new_warehouse = self.Warehouse.search([("company_id", "=", test_company.id)])
         test_picking_types = new_warehouse.get_picking_types()
         self.assertEqual(len(test_picking_types), 3)
+    
+    def test_check_picking_types_prefix_after_updating_warehouse_name(self):
+        """
+        Test that making changes to the warehouse does not cause the pickingtype prefixes to reset. 
```

Comment:
s/pickingtype/picking type/

Reply:
and: to reset -> to be reset

Reply:
Yes - the notation is from tools like [sed](https://en.wikipedia.org/wiki/Sed), it means

> substitute pickingtype with "picking type"

Reply:
okay cool :)